.github/workflows/main.yml

```diff
@@ -112,7 +112,8 @@ jobs:
     - uses: actions/setup-node@v2
       with:
         node-version: '16'
-    - run: cargo test
+    # Run everything except the example tests; we run them later in `test_examples`.
+    - run: cargo test --test must_use --test non_wasm --test std-crate-no-std-dep --test unwrap_throw
```

Comment:
Could testing of examples have something like `required-features = ['...']` to avoid the need to allow-list tests here? Otherwise this makes it unfortunately easy to forget to add a line here if a new test is added.